clusterloader2/api/types.go

```diff
@@ -20,6 +20,21 @@ import (
 	"time"
 )
 
+// TestSuite defines list of test scenarios to be run.
+type TestSuite []TestScenario
+
+// TestScenario defines customized test to be run.
+type TestScenario struct {
+	// Identifier is a unique test scenario name across test suite.
+	Identifier string `json: identifier`
+	// ConfigPath defines path to the config file that satisfy already
+	// existing cluster loader API.
```

Comment:
What is "existing cluster loader API"?

It should mention something like that this file contains a single Config.

Reply:
Done.